tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -1949,34 +1949,43 @@ def __init__(self, num_units, initializer=None, num_proj=None,
         projection matrices.
       num_proj: (optional) int, The output dimensionality for the projection
         matrices.  If None, no projection is performed.
-      number_of_groups: (optional) int, number of groups to use. If number_of_groups=1,
-        then it should be equivalent to LSTMP cell
+      number_of_groups: (optional) int, number of groups to use.
+        If number_of_groups=1, then it should be equivalent to LSTMP cell
```

Comment:
If `number_of_groups = 1`, then it should be equivalent to an LSTM cell.

(typo on LSTMP and use backticks around code)